scripts/zfs-probes.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/sh
```

Comment:
We probably want a license/copyright message here.

Reply:
Check.